components/admin_console/__snapshots__/admin_console.test.jsx.snap

```diff
@@ -18,138 +18,138 @@ exports[`components/AdminConsole should generate the routes 1`] = `
     >
       <Switch>
         <Route
-          key="license"
-          path="/license"
+          key="about/license"
+          path="/about/license"
```

Comment:
We will be breaking existing urls if we don't add redirection for old URl's to the new ones.

Ex: If we have old urls in posts they wont be working anymore. Are we ok with this?.

Reply:
We're generally OK with that. I don't think there's any places in the documentation where we link to specific system console pages. The fact is the whole page is getting re-organized so some of the old links won't work and that's ok as long as we update any links inside the product to point to the appropriate pages.

cc @wiersgallak

Reply:
I meant more in the lines of like plugins which use these URL's(I remember hearing NPS plugin using the url) or in existing posts in channel with the old URL's.

Reply:
Never mind. The chances of anyone depending on these URL's are really slim.